packs/ac-ancestries-and-class/Durian_Crab_W38aq5Oqel82AZ0w.json

```diff
@@ -39,6 +39,12 @@
         "mode": "add",
         "path": "system.skills.stealth.rank"
       },
+      {
+        "key": "ActiveEffectLike",
+        "value": -4,
+        "mode": "add",
+        "path": "system.abilities.int.mod"
+      },
```

Comment:
Isn't this on the class now?

Reply:
It's not - They're populating as 0
Unless I have an old item

Reply:
@pedrogrullada didn't you change that?

Reply:
It's the other way around, I moved them to the ancestries *from* the class, since they are not *always* –4